.github/workflows/p2p-workflow-extended-test.yaml

```diff
@@ -33,14 +29,37 @@ on:
         required: false
         type: string
         default: '.'
+      registry-path:
+        required: false
+        type: string
+        default: 'extended-test'
+      image-name:
+        required: true
+        type: string
 
 env:
   REGION: ${{ inputs.region }} 
 
 jobs:
+  get-latest-version:
```

Comment:
i don't think this can be here, as we don't mandate that the version comes from a registry

i think we should make it like fast feedback where we expect everyone to do the same thing but its still called from the outside